businesscentral-indexing/dev-overrides.yaml

```diff
@@ -0,0 +1,17 @@
+schema_version: 1
+
+from: "rhpam-7/rhpam71-businesscentral-indexing:latest"
```

Comment:
As the indexing module changed, shouldn't the image name change to workbench-indexing too?

Reply:
No because the product image here is still businesscentral. The reasons the script module changed name is 1) because it's shared between both businesscentral and decisioncentral, and I needed a different name, 2) this work will lead us to community images, where it is actually called workbench.

Reply:
+1